purchase_propagate_qty/models/purchase.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2014-2016 Numérigraphe SARL
+# Copyright 2017 Eficent Business and IT Consulting Services, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import logging
+from itertools import groupby
+from odoo import models, api, fields
+from odoo.tools import float_is_zero
+
+_logger = logging.getLogger(__name__)
+
+
+class PurchaseOrderLine(models.Model):
+    _inherit = "purchase.order.line"
+
+    @api.multi
+    def write(self, values):
+        res = super().write(values)
```

Comment:
Couldn't it be possible to get rid of that local variable since it's not used in the method ?

Reply:
which one? `res` ? I return it at the end of the method (on line 32)

Reply:
Yes, I mean, why not directly return super().write(values) ?

Reply:
It's not the same algorithm. (Not the same order)